CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ Note that Sockeye has checks in place to not translate with an old model that wa
 
 Each version section may have have subsections for: _Added_, _Changed_, _Removed_, _Deprecated_, and _Fixed_.
 
+## [1.18.6]
+- Added support for LHUC
```

Comment:
We could list the paper title and link here, plus a note that this is currently for RNNs.